### ISSUE TITLE
TypeError: adapter is not a constructor

### ISSUE THREAD

**Issue by benfrancis** [moziot/gateway#12]
STR:
* Run "node test-server.js"

Get error:

```tola@trelkez:~/Code/moziot/gateway$ node test-server.js 
Listening at http://:::8088
Loading adapter ./adapters/foo.js
/home/tola/Code/moziot/gateway/adapters.js:43
                adapter_manager.adapters.push(new adapter(adapter_manager));
                                              ^

TypeError: adapter is not a constructor
    at file_list (/home/tola/Code/moziot/gateway/adapters.js:43:47)
    at FSReqWrap.oncomplete (fs.js:123:15)
```

@dhylands Do you not see this error? Are you running test-server.js in a different way?

**Comment by dhylands** [comment]
Yes - I ran into this yesterday and have a fix. I'm not sure why I didn't run into it before my PR.

**Comment by dhylands** [comment]
This should be fixed by #13

**Comment by benfrancis** [comment]
Thanks, I should have caught this in review (by you know, actually running the code!).
